repair-service/service-api/src/main/java/com/epam/repair/rest_app/CityService.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.repair.rest_app;
```

Comment:
btw
why service is in `rest_app` package?
thing it should be in a separate package

Reply:
Comment accepted. Package name changed.